core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperatorTableFactory.java

```diff
@@ -137,10 +139,21 @@ private static boolean operatorIsInLibrary(String operatorName, Field field,
       throw new AssertionError("Operator must belong to at least one library: "
           + operatorName);
     }
+    final Set<SqlLibrary> excludeLibrarySet = new HashSet<>();
```

Comment:
fold these two lines into one.

use `ImmutableSet.of` (which, I believe, optimizes for enums)

Reply:
Updated.